content/reference/host-runtime/security.md

```diff
@@ -0,0 +1,79 @@
+    "tags": [],
+    "caps": [
+      "wasmcloud:keyvalue",
+      "wasmcloud:http_server"
+    ],
+    "prov": false
+  }
+}
+```
+
+Here one of the most important fields in the `wascap` object is the `caps` field, which contains an array of _capability contract IDs_. Each capability provider must conform to a single, globally unique contract identifier. This ID is typically prefixed with a namespace or vendor prefix, but is not a requirement.
+
+The preceding token indicates that the _Key Value Counter_ actor has been granted access to the Key-Value and HTTP Server capabilities, _without regard_ for which specific provider is used to satisfy those capabilities. In other words, this actor can use _any_ key-value provider, be it Redis or Cassandra or Consul. If you want to further limit which capabilities can be used by actors at runtime, you can [implement an authorizer plugin](/platform-builder/authorizer).
+
+### Managing Keys
+
+The act of key management could be a book or tome all its own. In short, managing keys in wasmCloud involves managing the 57-character plain ASCII strings that represent the developer-friendly encoding of ed25519 keys and identities we use. The entire security system of wasmCloud is designed around the idea that, while running in a production (or any) environment, _no private key access is required_. You will never need to expose a private key to any portion of your running environment.
+
+That said, you will need to ensure that offline access to your keys is secured. Your choice for how you do that is entirely up to you and the needs of your organization.
+
+#### Key Types
+
+wasmCloud uses a special, user-friendly encoding for ed25519 keys that is identical to the encoding used by [NATS](https://nats.io) for its 2.0+ decentralized security mechanism. This encoding uses all uppercase letters with an **S** prefix for seeds followed by a key type prefix. An underrated benefit of this key encoding mechanism is that keys are double-clickable on most operating systems and a fixed length.
+
+While the number of key encodings available is larger than this list, the following are key types that may be of interest to wasmCloud developers:
+
+* Actor (**M**odule) - Produces public keys with the **M** prefix.
+* **A**ccount - Produces public keys with the **A** prefix.
+* Capability Pro**v**ider (Ser**v**ice) - Produces public keys with the **V** prefix.
+* Server/**N**ode - Produces public keys with the **N** (node) prefix.
+* **O**perator - Produces public keys with the **O** prefix.
+* **C**luster - Produces public keys with the **C** prefix.
+
+As mentioned, _by convention_, actor tokens are issued by accounts. **A**ccounts are, by convention, issued by operators. Ser**v**ices (capability pro**v**iders) are also issued/signed by accounts. Servers (wasmCloud processes/**n**odes) are currently self-issued/self-signed, though in the future they could potentially be issued by a cluster to provide additional levels of security.
```

Comment:
Big fan of the bolding here. Drives the point home that these keys are easily human identifiable